### PR TITLE
Rewrite SC10 shutter enable as toggle

### PR DESCRIPTION
**Pull request by trappitsch**
As discussed in #368, this PR rewrites the `sc.enable` routine for the Thorlabs SC10 shutter as a toggle. The toggle is implemented such that any setting of the property first queries the current status and then acts accordingly, i.e., toggle the status if necessary.
Tests are implemented for all cases. Existing get / set property test was split out into a get and set test. The latter now tests for cases, such that all cases are covered.

### CODE REVIEW

File: src/instruments/thorlabs/sc10.py [hunk 1]
```diff
@@ -64,20 +64,27 @@ def name(self):
...
+        :raises TypeError: Unexpected type given when trying to enable.
+        """
+        return bool(int(self.query("ens?")))
+
+    @enable.setter
+    def enable(self, value):
+        if not isinstance(value, bool):
+            raise TypeError(f"Expected bool, got type {type(value)} instead.")
+        curr_status = self.enable
+        if curr_status != value:
```
Comment (scasagrande) [inline src/instruments/thorlabs/sc10.py:86]: when comparing `bool` we should use `is` and `is not`

so I would just change this to be `if curr_status is not value:`

**Review by scasagrande — CHANGES_REQUESTED** [review]
one small comment but then looks good

File: src/instruments/thorlabs/sc10.py (hunk 1 above)
Comment (trappitsch) [reply src/instruments/thorlabs/sc10.py]: Thanks, this is changed now

**Comment by trappitsch** [comment]
Oops, also my main branch was pretty badly outdated... should be fixed now. Sorry for that

**Pull request merged**